### PR TITLE
add support to rhel-87

### PR DESCRIPTION
**Pull request by acosferreira**
# Description

Add support to create images on RHEL-87 distribution

Fixes # (THEEDGE-2808)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `npm run lint:js:fix` to check that my code is properly formatted

### CODE REVIEW

File: src/constants/index.js [hunk 1]
```diff
@@ -64,17 +64,19 @@ export const distributionMapper = {
...
 export const releaseMapper = {
   'rhel-90': 'Red Hat Enterprise Linux (RHEL) 9.0',
+  'rhel-87': 'Red Hat Enterprise Linux (RHEL) 8.7',
   'rhel-86': 'Red Hat Enterprise Linux (RHEL) 8.6',
   'rhel-85': 'Red Hat Enterprise Linux (RHEL) 8.5',
   'rhel-84': 'Red Hat Enterprise Linux (RHEL) 8.4',
 };
 
-export const supportedReleases = ['rhel-84', 'rhel-85'];
+export const supportedReleases = ['rhel-84', 'rhel-85', 'rhel-87'];
```
Comment (omaciel) [inline src/constants/index.js:79]: What does `supportedReleases` mean? We should be supporting both `rhel-86` and `rhel-90` right now. Also, if `temporaryReleases` below is a way to track releases that we're not fully supporting yet, then we could add `reel-87` there until GA?

**Review by omaciel — COMMENTED** [review]
I need some clarification.

File: src/constants/index.js (hunk 1 above)
Comment (matoval) [reply src/constants/index.js:79]: I believe supportedReleases does mean releases we are officially supporting and temporyReleases are ones we are testing and will officially support soon.
@ldjebran Is that correct?

Comment (acosferreira) [reply src/constants/index.js:79]: @omaciel per code read, we use the temporary and default to enable or disable the versions based on the feature flag.

Comment (acosferreira) [reply src/constants/index.js:79]: I believe we can move all to default and add only 91 to temporary given is not in ga

**Review by matoval — APPROVED** [review]

**Review by de1987 — APPROVED** [review]

**Pull request merged**